docs/topics/whatsnew-eap.md

```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
 
 ## New Kotlin/Wasm target
 
-> The new backend is available in EAP of IntelliJ IDEA 2022.3 only.
->
-{type="note"}
+Kotlin/Wasm (Kotlin WebAssembly) goes [Experimental](components-stability.md#stability-levels-explained) in this preview release!
+Kotlin team finds [WebAssembly](https://webassembly.org/) is promising technology. We have a close look at it and 
+try to find better ways how you can use WebAssembly and get all the benefits of Kotlin.
+
+WebAssembly binary format is independent of the platform because it runs using its own virtual machine.
+Almost all the modern browsers support WebAssembly in an experimental mode, so you don't need to install additional tooling to 
+have an environment to run WebAssembly, therefore, Kotlin/Wasm.
```

Comment:
[The GC proposal](https://github.com/WebAssembly/gc) we rely on is still experimental and hidden under the flag.

Reply:
@koshachy important to fix!